crates/virtio-queue/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
-# Unreleased
+# v0.3.0
 
 ## Added
-- QueueStateT::size(): [#153](https://github.com/rust-vmm/vm-virtio/pull/153)
+- Improve the usability of queue interfaces: [#148](https://github.com/rust-vmm/vm-virtio/pull/148)
```

Comment:
Can we somehow summarize the exact API updates (e.g Added QueueStateOwnedT trait which does x/can be used for; Added pop_descriptor_chain method to y for doing z. etc)? The `improve ....` is very vague, and not necessarily useful in any way.

Reply:
Done, I summarized the new methods as in the other entries, and mentioned what the new interface is for.